docscan/session/create/filter/document_restrictions_filter.go

```diff
@@ -16,20 +17,23 @@ func (r RequestedDocumentRestrictionsFilter) Type() string {
 // MarshalJSON returns the JSON encoding
 func (r *RequestedDocumentRestrictionsFilter) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&struct {
-		Type      string                          `json:"type"`
-		Inclusion string                          `json:"inclusion"`
-		Documents []*RequestedDocumentRestriction `json:"documents"`
+		Type                  string                          `json:"type"`
+		Inclusion             string                          `json:"inclusion"`
+		Documents             []*RequestedDocumentRestriction `json:"documents"`
+		AllowExpiredDocuments *bool                           `json:"allow_expired_documents,omitempty"`
```

Comment:
what is the purpose of using both `*bool` and `omitempty`?

Reply:
Thats resolved with commit